content/references/http-websocket-apis/public-api-methods/server-info-methods/server_state.md

```diff
@@ -306,6 +310,11 @@ The `state` object may have some arrangement of the following fields:
 | `validation_quorum`              | Number          | Minimum number of trusted validations required to validate a ledger version. Some circumstances may cause the server to require more validations. |
 | `validator_list_expires`         | Number          | _(Admin only)_ When the current validator list will expire, in [seconds since the Ripple Epoch][], or 0 if the server has yet to load a published validator list. [New in: rippled 0.80.1][] |
 
+[Reporting mode]: rippled-server-modes.html
```

Comment:
duplicate?

Reply:
Not a duplicate. These reference links are case-sensitive, and only apply in the file where they're defined (except when using includes, which is not the case for this one).

The uppercase-R version is used on 292-295 and the lowercase-r version is used on line 289.